services/course-material/src/components/layout/DynamicSvg.tsx

```
@@ -0,0 +1,51 @@
+    <div
+      className={css`
+        path {
+          fill: ${baseTheme.colors.gray[500]};
+        }
+
+        svg {
+          width: 5rem;
+          height: 5rem;
+        }
+      `}
+      dangerouslySetInnerHTML={{ __html: svgContent }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)